src/analyticalSolutions/sedovSolution/main.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/*! @file
+ *
+ * @brief This program generate the analytical sedov solution based in the time and the initial conditions
+ *
+ * @author Jose A. Escartin <ja.escartin@gmail.com>
+ *
```

Comment:
You may want to add a reference:
https://cococubed.com/papers/kamm_2000.pdf

Reply:
Hi, yes sure, but this is already added but not in the 'main.ccp' where only is called. The reference is in the solution header itself 'SedovSolution.hpp'

 *        This routine is a C++ conversion of one Fortran code based in these two papers:
 *        - "evaluation of the sedov-von neumann-taylor blast wave solution", Jim Kamm, la-ur-00-6055
 *        - "the sedov self-similiar point blast solutions in nonuniform media", David Book, shock waves, 4, 1, 1994

Reply:
The Frank Timmes code (in the source that you pointed out) is a evolution, wrote in 2019, of the base code that I used.
https://cococubed.com/research_pages/sedov.shtml

Reply:
I ported and removed the bad practices (as goto jumps!) but maybe need a more deep refactoring to be more nice and readable.

Reply:
Checking the results in 1D, 2D and 3D, both codes generate the same outputs but with a small diferences done by the numerical noise introduced for the conversion from Fortran to C++

Reply:
Checking the results in 1D, 2D and 3D, both codes generate the same outputs but with a small differences done by the numerical noise introduced for the conversion from Fortran to C++